packages/ui/lib/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { ethers } from 'ethers'
+
+export const minifyAddress = (address: string) => {
+  const checked = ethers.utils.isAddress(address)
+  return checked
+    ? `${address.slice(0, 5)}...${address.slice(address.length - 5)}`
```

Comment:
```suggestion
  return checked && address.length > 5
    ? `${address.slice(0, 5)}...${address.slice(address.length - 5)}`
```

Reply:
No, this is incorrect because this is only for letting ENS name pass through as is instead of converting to address. This will stop any ENS less than 5 in length to be passed.